### PR TITLE
docs: Added Bedrock embeddings examples

### PR DESCRIPTION
**Pull request by Anush008**
[PREVIEW](https://deploy-preview-518--condescending-goldwasser-91acf0.netlify.app/documentation/embeddings/bedrock/)

### CODE REVIEW

File: qdrant-landing/content/documentation/embeddings/bedrock.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+weight: 1000
+---
+
+# Bedrock Embeddings
+
+Qdrant can be used with [AWS Bedrock](https://aws.amazon.com/bedrock/) which offers multiple [embedding model providers](https://docs.aws.amazon.com/bedrock/latest/userguide/models-supported.html) to choose from.
+
+To access the hosted embedding providers, authentication with your AWS credentials is required. Your can learn how to obtain and configure your credentials [here](https://repost.aws/knowledge-center/create-access-key)."
+
+Below is an example to generate embeddings using `Titan Embeddings G1 - Text` model that produces sentence embeddings of size 1536 and add them to Qdrant.
```
Comment (mjang) [inline qdrant-landing/content/documentation/embeddings/bedrock.md:12]: I suggest focusing more on the user, with "second person (you)" and active voice (avoid verbs like "is"). I've also focused on the info the reader needs to add to the code samples you've provided.

Also -- I linked to https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html -- and I _assume_ that "size 1536" depends on AWS Bedrock. (I don't want us to guarantee that kind of performance.)

Note: I have not checked your code samples.

```suggestion
You can use [AWS Bedrock](https://aws.amazon.com/bedrock/) with Qdrant. AWS Bedrock supports multiple [embedding model providers](https://docs.aws.amazon.com/bedrock/latest/userguide/models-supported.html).

You'll need the following information from your AWS account:

- Region
- Access key ID
- Secret key

To configure your credentials, review the following AWS article: [How do I create an AWS access key](https://repost.aws/knowledge-center/create-access-key)."

With the following code sample, you can generate embeddings using the [Titan Embeddings G1 - Text model](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html). Amazon suggests that it produces sentence embeddings of size 1536.
```

**Review by mjang — CHANGES_REQUESTED** [review]
See comment

File: qdrant-landing/content/documentation/embeddings/bedrock.md (hunk 1 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/embeddings/bedrock.md]: The embeddings size of the model is guaranteed to not change.

Comment (Anush008) [reply qdrant-landing/content/documentation/embeddings/bedrock.md]: https://github.com/qdrant/landing_page/commit/7fff147a4f63d33cd7a7d5513ef7dff66ce19c7b

**Pull request merged**